server/src/test/java/org/elasticsearch/action/admin/indices/create/ShrinkIndexIT.java

```diff
@@ -580,7 +580,9 @@ public void testShrinkThenSplitWithFailedNode() throws Exception {
             .build()).setResizeType(ResizeType.SHRINK).get());
         ensureGreen();
 
+        final int nodeCount = cluster().size();
         internalCluster().stopRandomNode(InternalTestCluster.nameFilter(shrinkNode));
+        ensureStableCluster(nodeCount -1);
```

Comment:
NIT: formatting of the `-1` is missing a space `- 1` :)

Reply:
@original-brownbear you've got eagle :eyes: :)